cromwell-backend/src/main/scala/cromwell/backend/model/TaskStatus.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.backend.model
+
+import cromwell.backend.model.Status.Status
+
+/**
+  * Represents status of an execution.
+  */
+object Status extends Enumeration {
+  type Status = Value
+  val Created, Running, Succeeded, Failed, Canceled = Value
+}
+
+/**
+  * Defines a task status with its result data.
+  */
+case class TaskStatus(status: Status, result: Option[ExecutionResult]) extends ExecutionEvent
```

Comment:
I know I should create a trait for TaskStatus for this... but I think is better as it's now because is cleaner.